client/src/pages/HomePage.jsx

```diff
@@ -1,55 +1,25 @@
-import { Grid, Typography, useTheme } from "@mui/material";
 import React from "react";
+import { Grid, Stack } from "@mui/material";
 
 import BasePage from "./BasePage";
+import HomePageContent from "../components/HomePageContent";
+import SearchBar from "../components/SearchBar";
 
 /**
  * https://github.com/BobaTalks/internship-tracker/issues/24
  * Will need to include the background styles for the HomePage per design spec
  */
 const HomePage = (props) => {
-  const theme = useTheme();
   return (
     <BasePage>
-      <Grid
-        container
-        direction="column"
-        justifyContent="center"
-        alignItems="flex-start"
-        sx={{ height: "100vh" }}
-        spacing={0}
-      >
-        <Grid item>
-          <Typography
-            variant="title"
-            sx={{
-              [theme.breakpoints.down("sm")]: {
-                fontSize: "3rem",
-                lineHeight: "3rem",
-              },
-            }}
-          >
-            Personalized internship tracking made easy.
-          </Typography>
-        </Grid>
-        <Grid item container direction="row" marginTop={2}>
-          <Grid item sm={7} xs={12}>
-            <Typography
-              variant="body1"
-              sx={{
-                [theme.breakpoints.down("sm")]: {
-                  fontSize: "1rem",
-                },
-              }}
-            >
-              Track when internships open for the company of your choice. Find
-              internships for all the roles you want. Manually track/update your
-              application statuses.
-            </Typography>
+      <Stack spacing={5}>
+        <HomePageContent />
+        <Grid container justifyContent="center">
+          <Grid containter item lg={8} md={9} xs={11}>
```

Comment:
Is this supposed to be a container?

Reply:
I believe so since it contains the Search Bar, which in itself, contains other grids - correct me if i'm wrong

Reply:
I wouldn't say wrong, we can leave it for now!

Reply:
I talked to a friend about this and I think you're right about it...I will change it